src/org/openstreetmap/josm/plugins/osmobjinfo/OSMObjInfoActions.java

```diff
@@ -45,11 +53,11 @@ static void openinBrowserUserOsmComments(String user) {
 
     public static void copyChangeset(String idChangeset) {
         if (!idChangeset.isEmpty()) {
-            String linkchangeset = "https://www.openstreetmap.org/changeset/" + idChangeset;
+            String linkchangeset = "https://www.openstreetmap.org/changeset/".concat(idChangeset);
```

Comment:
why this change?

Reply:
I think one of my lint tools was complaining about it (Sonarlint maybe?)